preinstall.js

```diff
@@ -59,13 +59,49 @@ switch (os.platform()) {
     break
 }
 
+function findMsBuild () {
+  var programmFiles
+
+  if (arch === 'x64') {
+    programmFiles = process.env['PROGRAMFILES(X86)']
+  } else {
+    programmFiles = process.env['PROGRAMFILES']
+  }
+
+  var possiblePaths = [
+    programmFiles + '/Microsoft Visual Studio/2017/BuildTools/MSBuild/15.0/Bin/msbuild.exe',
+    programmFiles + '/Microsoft Visual Studio/2017/Enterprise/MSBuild/15.0/Bin/msbuild.exe',
+    programmFiles + '/Microsoft Visual Studio/2017/Professional/MSBuild/15.0/Bin/msbuild.exe',
+    programmFiles + '/Microsoft Visual Studio/2017/Community/MSBuild/15.0/Bin/msbuild.exe',
+    programmFiles + '/MSBuild/14.0/Bin/MSBuild.exe',
+    programmFiles + '/MSBuild/12.0/Bin/MSBuild.exe',
+    process.env.WINDIR + '/Microsoft.NET/Framework/v4.0.30319/MSBuild.exe'
+  ]
+
+  for (var counter = 0; counter < possiblePaths.length; counter++) {
+    var possiblePath = possiblePaths[counter]
+
+    try {
+      fs.accessSync(path.resolve(possiblePath), fs.constants.R_OK | fs.constants.X_OK)
```

Comment:
Seems `X_OK` is noop on windows: https://nodejs.org/api/fs.html#fs_file_access_constants

Reply:
I'll ditch all the parameters, since `F_OK` is the default and seems to be ok.